src/steamship/utils/huggingface_helper.py

```diff
@@ -27,38 +27,34 @@ async def _model_call(session, text: str, api_url, headers, additional_params: d
     if it believes you have 'too many' requests simultaneously, so the logic retries in this case, but fails on
     other errors.
     """
-    while True:
-        tries = 0
-        async with session.post(api_url, headers=headers, data=data) as response:
+    n_tries = 0
+    while n_tries <= max_error_retries:
+        async with session.post(api_url, headers=headers, json=json_input) as response:
             if response.status == HTTPStatus.OK and response.content_type == "application/json":
-                json_response = await response.json()
-                logging.info(json_response)
-                return json_response
+                ok_response = await response.json()
+                logging.info(ok_response)
+                return ok_response
             else:
-                text_response = await response.text()
-                if "is currently loading" not in text_response:
+                nok_response = await response.text()
+                if "is currently loading" not in nok_response:
                     logging.info(
-                        f"received text response [{text_response}] for input text [{text}], attempt {tries}"
+                        f'Received text response "{text_response}" for input text "{text}" [attempt {n_tries}/{max_error_retries}]'
                     )
-                    if tries >= max_error_retries:
-                        raise SteamshipError(
-                            message="Unable to query Hugging Face model",
-                            internal_message=f"HF returned error: {text_response} after {tries} attempts",
-                        )
-                    else:
-                        tries += 1
+                    n_tries += 1
                 else:
                     await asyncio.sleep(1)
+    if ok_response is None:
+        raise SteamshipError(
+            message="Unable to query Hugging Face model",
+            internal_message=f"HF returned error: {text_response} after {n_tries} attempts",
+        )
+    return ok_response
 
 
 async def _model_calls(
-    texts: List[str],
-    api_url: str,
-    headers,
-    timeout_seconds: int,
-    additional_params: dict = None,
+    texts: List[str], api_url: str, headers, additional_params: dict = None
 ) -> List[list]:
-    async with aiohttp.ClientSession(timeout=ClientTimeout(total=timeout_seconds)) as session:
+    async with aiohttp.ClientSession(timeout=ClientTimeout(total=10)) as session:
```

Comment:
It looks like you accidentally undid the timeout_seconds change in a merge conflict.